### PR TITLE
[CDAP-14698] Changes 'directives' tab name to 'Transformation steps' …

### PR DESCRIPTION
**Pull request by ajainarayanan**
…in dataprep

Same PR but for 6.0 https://github.com/cdapio/cdap/pull/11018

Build: https://builds.cask.co/browse/CDAP-UDUT274

### CODE REVIEW

**Comment by ajainarayanan** [comment]
Waiting on review from @bdmogal

**Review by yukiej — APPROVED** [review]

**Comment by ajainarayanan** [comment]
Presented to product (@bdmogal). Merging.

**Comment by ajainarayanan** [comment]
Latest build: https://builds.cask.co/browse/CDAP-UDUT274-3

**Pull request merged**
